packages/modules/vehicles/smarteq/api.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+
+import logging
+from typing import Union
+import asyncio
+import json
+from modules.common.store import RAMDISK_PATH
+from modules.vehicles.smarteq.config import SmartEQ
+import requests
+import bs4
+import pkce
+import pickle
+
+date_fmt = '%Y-%m-%d %H:%M:%S'
+# refreshToken_exp_days = 7    # 7 days before refreshToken expires a new refreshToken shall be stored
+initialToken = '1.2.3'
+
+log = logging.getLogger("soc."+__name__)
+
+# Constants
+BASE_URL = "https://id.mercedes-benz.com"
+OAUTH_URL = BASE_URL + "/as/authorization.oauth2"
+LOGIN_URL = BASE_URL + "/ciam/auth/login"
+TOKEN_URL = BASE_URL + "/as/token.oauth2"
+STATUS_URL = "https://oneapp.microservice.smart.com"
+REDIRECT_URI = STATUS_URL
+SCOPE = "openid+profile+email+phone+ciam-uid+offline_access"
+CLIENT_ID = "70d89501-938c-4bec-82d0-6abb550b0825"
+GUID = "280C6B55-F179-4428-88B6-E0CCF5C22A7C"
+ACCEPT_LANGUAGE = "de-de"
+
+
+class api:
```

Comment:
Bitte am Python-Style-Guide orientieren: Klassen-Namen beginnen mit einem Großbuchstaben, Variablen und Attribute mit Kleinbuchstaben.
```suggestion
class Api:
```